flight/Engine.php

```diff
@@ -481,18 +499,14 @@ public function _start(): void
                 $params
             );
 
-            if ($response->v2_output_buffering === false) {
+            if ($response->v2_output_buffering === false && $route->is_streamed === false) {
```

Comment:
You could extract that conditional in a separate method to avoid duplications

Reply:
How about I'll pull it out to a variable. That'll be easiest since it's really only used in this one method